src/rpc/format/cbor/convert.rs

```diff
@@ -54,23 +56,45 @@ impl TryFrom<Cbor> for Value {
 			Data::Tag(t, v) => {
 				match t {
 					// A literal datetime
-					TAG_DATETIME => match *v {
+					TAG_STRING_DATETIME => match *v {
 						Data::Text(v) => match Datetime::try_from(v) {
 							Ok(v) => Ok(v.into()),
 							_ => Err("Expected a valid Datetime value"),
 						},
 						_ => Err("Expected a CBOR text data type"),
 					},
+					// An epoch-based datetime (https://www.rfc-editor.org/rfc/rfc8949.html#name-epoch-based-date-time)
+					TAG_DATETIME => match *v {
+						Data::Integer(v) => match Datetime::try_from(i128::from(v)) {
+							Ok(v) => Ok(v.into()),
+							_ => Err("Expected a valid Datetime value"),
+						},
+						Data::Float(v) => match Datetime::try_from(v) {
```

Comment:
Does the spec also allow decimals values or just a float?

Reply:
![image](https://github.com/surrealdb/surrealdb/assets/6053067/d041b2c0-7464-4a54-8796-7a3b4df7539b)

Integer and float, that means no decimal if I understand it clearly.